app/controllers/api/v2/moves_actions.rb

```diff
@@ -62,10 +62,15 @@ def move_params
     end
 
     def move_attributes
-      move_params[:attributes].tap do |attributes|
-        attributes[:profile] = profile unless profile_attributes.nil?
+      update_move_attributes.tap do |attributes|
         attributes[:from_location] = from_location unless from_location_attributes.nil?
         attributes[:to_location] = to_location unless to_location_attributes.nil?
+      end
+    end
+
+    def update_move_attributes
```

Comment:
how about `def common_move_attributes` ?

Reply:
yes, I agree 👍

Reply:
Good idea, I like that too 👍